wzdx/sample_files/enhanced/version_4_1.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
File doesn't seem to be referenced anywhere.

Reply:
removed